src/pages/Main/Main.js

```diff
@@ -1,7 +1,90 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Slide from './Slide';
+import Category from './Category';
+import { CATEGORY } from './CategoryListData';
 
-function Main() {
-  return <div>main</div>;
-}
+const Main = () => {
+  const [sliders, setSliders] = useState([]);
+  const [selectedCategory, setSelectedCategory] = useState(1);
+
+  useEffect(() => {
+    fetch(`http://10.58.0.252:8000/${selectedCategory}`)
```

Comment:
```suggestion
    fetch(`${API}/${selectedCategory}`)
```
ip랑 포트 번호는 src/config.js로 분리해주세요